src/fluree/db/dbfunctions/fns.cljc

```diff
@@ -313,9 +326,9 @@
       (raise ?ctx "Cannot access ?pO from this function interface"))))
 
 (defn get-all
-  {:doc      "Follows an subject down the provided path and returns a set of all matching subjects."
+  {:doc      "Used to get-all values in a nested result set, or also can follow an subject down the provided path and returns a set of all matching subjects."
```

Comment:
Not your typo, but since you touched the doc string: s/an/a/ in "an subject"

Also: s/returns/return/ in "returns a set"